### ISSUE TITLE
Richtext plugin

### ISSUE THREAD

**Issue by onno-timmerman** [wearebraid/vue-formulate#395]
It would be very superb nice to also have simple richtext area type. I think this might be a plugin with tiptap as base.
https://tiptap.dev/

**Comment by justin-schroeder** [comment]
Absolutely. Vue Formulate core will only ever have "native" inputs, but we encourage people to write and share plugins for these "sythetic" input types. I know @mcopper1226 actually already wrote a tiptap version, but we'll probably wait till version 3 to release our own. Still, if someone wants to publish one we're always happy to post it on the [plugins page](https://vueformulate.com/guide/plugins/).
